platform/mbed-trace/source/CMakeLists.txt

```diff
@@ -1,18 +1,18 @@
 if(DEFINED TARGET_LIKE_X86_LINUX_NATIVE)
-    add_library( mbed-trace
-        mbed_trace.c
-    )
+    # add_library( mbed-trace
```

Comment:
Would it be worth explaining, in a comment, why all these directives are commented out?